0-module-and-module-source.emu

```diff
@@ -0,0 +1,614 @@
+<!doctype html>
+<html lang="en-GB-oxendict">
+<head>
+<meta charset="utf8">
+<link href="ecmarkup.css" rel="stylesheet">
+<script src="ecmarkup.js"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/solarized-light.min.css">
+</head>
+
+<body>
+<pre class="metadata">
+title: 'Module & ModuleSource API'
+stage: 1
+contributors: Mark Miller, Caridy Patiño, Patrick Soquet, Kris Kowal, Jack Works, Guy Bedford
+status: proposal
+copyright: false
+location: https://tc39.es/proposal-compartments/
+</pre>
+
+<emu-clause id="sec-well-known-intrinsic-objects">
+    <h1>Well-known intrinsic objects</h1>
+    <emu-table id="table-7" caption="Well-known Intrinsic Objects">
+        <table>
+            <tbody>
+                <tr>
+                    <th>Intrinsic Name</th>
+                    <th>Global Name</th>
+                    <th>ECMAScript Language Association</th>
+                </tr>
+                <tr>
+                    <td>%ModuleSource%</td>
+                    <td>`ModuleSource`</td>
+                    <td>The ModuleSource constructor (<emu-xref href="#sec-modulesource-constructor"></emu-xref>)</td>
+                </tr>
+                <tr>
+                    <td>%Module%</td>
+                    <td>`Module`</td>
+                    <td>The Module constructor (<emu-xref href="#sec-module-constructor"></emu-xref>)</td>
+                </tr>
+            </tbody>
+        </table>
+    </emu-table>
+</emu-clause>
+
+<emu-clause id="sec-modulesource-records">
+    <h1>Module Source Records</h1>
+
+    <p>A <dfn id="modulesource-record" variants="Module Source Records">Module Source Record</dfn> is used to represent information about a module source that was defined from ECMAScript source text (<emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>) that was parsed using the goal symbol |Module|. Its fields contain digested information about the names that are imported and exported by the module.</p>
```

Comment:
```html
(<emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>)
```

this part is rendered as `()` in html.

Reply:
thanks for the help, biblio was missing.